src/core/server/opensearch/client/client_config.ts

```diff
@@ -115,6 +116,10 @@ export function parseClientOptions(config: OpenSearchClientConfig, scoped: boole
     );
   }
 
+  if (config.disablePrototypePoisoningProtection != null) {
+    clientOptions.disablePrototypePoisoningProtection = config.disablePrototypePoisoningProtection;
```

Comment:
How would this handle "false" as a string?

Reply:
Good call out. Right now it's defined as optional or boolean so not able to pass in a string without failing to bootstrap.